cider/exec/module/batch/CiderBatchUtils.cpp

```diff
@@ -43,9 +43,9 @@ namespace CiderBatchUtils {
     C_TYPE* null_buf = (C_TYPE*)table_ptr[i * 2];                           \
     for (auto j = 0; j < table_row_num; ++j) {                              \
       C_TYPE value = buf[j];                                                \
-      bool is_null = null_buf[j];                                           \
+      bool is_not_null = null_buf[j];                                       \
```

Comment:
Hmm, why do we need to create ```is_not_null``` variable? Just set value to ```null_data``` directly.

Reply:
should convert null sign value from C_TYPE to bool.